src/main/java/org/opensearch/performanceanalyzer/collectors/SearchBackPressureStatsCollector.java

```diff
@@ -320,12 +332,17 @@ public SearchTaskStats(
                                 SearchBackPressureStatsValue.Constants
                                         .SEARCHBP_SEARCH_TASK_STATS_LIMITREACHEDCOUNT)
                         long limitReachedCount,
+                @JsonProperty(
+                                SearchBackPressureStatsValue.Constants
+                                        .SEARCHBP_SEARCH_TASK_STATS_COMPLETIONCOUNT)
+                        long completionCount,
```

Comment:
Same as above.

Reply:
We already have the changes in place to prevent it.